articles.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+---
+
```

Comment:
I think we should have a title for the page.  All our usual second level pages have titles marked by `##`.